### PR TITLE
topic0-keypoint3-solution

### PR DESCRIPTION
**Pull request by leandrodalbo**
Hi, Let me know when I' ll be able to merge it. 
See you.

### CODE REVIEW

File: topic0keyPoint3/src/topic0keyPoint3/HospitalFactory.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package topic0keyPoint3;
+
+public class HospitalFactory {
+
+	public Hospital getAhospital(String hospitalType) {
+
+		if (hospitalType.equals("psychiatricHospital")) {
```
Comment (tomiok) [inline topic0keyPoint3/src/topic0keyPoint3/HospitalFactory.java]: Two things here:
if psychiatricHospital is null, could throw a null pointer exception. So the better way is invert the args
```
if ("psychiatricHospital".equalsIgnoreCase(hospitalType) {

}
```

Also see the method **equalsIgnoreCase()**, that ignore the cases...

**Comment by leandrodalbo** [comment]
Hi Tomi, the bug is already fixed, you tell me if it is ready to be merged.

**Comment by tomiok** [comment]
nice work, merge it!

**Comment by tomiok** [comment]
and close the issues either.

**Pull request merged**